ci/gdk_build_macos.template.steps.yaml

```diff
@@ -0,0 +1,36 @@
+agent_transients: &agent_transients
+  # This is designed to trap and retry failures because agent lost
+  # connection. Agent exits with -1 in this case.
+  exit_status: -1
```

Comment:
There's one for 255 too.

Reply:
what would I need to add for that?

Reply:
ahh, thanks! is this only for windows? or all platforms?